src/business.c

```diff
@@ -0,0 +1,368 @@
+#include "stdbool.h"
+#include "stdint.h"
+#include "business.h"
+#include "main.h"
+#include "os.h"
+#include "cx.h"
+#include "string.h"
```

Comment:
Nitpicking, but I prefer when toolchain includes are on top with `<>` and not `""`. Same in other files of this PR.